lib/active_merchant/billing/gateways/ixopay.rb

```diff
@@ -173,6 +176,17 @@ def add_preauth(xml, money, options)
         end
       end
 
+      def add_capture(xml, money, authorization, options)
+        currency = options[:currency] || currency(money)
+
+        xml.capture do
+          xml.transactionId          new_transaction_id
+          xml.referenceTransactionId authorization
+          xml.amount                 localized_amount(money, currency)
```

Comment:
Is calling `localized_amount` here necessary? If so, then should this also be used on all of the other transaction types?

Reply:
Yes to both, I believe. ```localized_amount``` gives you an amount in (for example) dollars, converting e.g. 100 to 1.00. That is the format specified in the docs, and I think we just missed this conversion initially. I'll update all of the other instances in this PR.

Reply:
Done.